csi/volumeutils.py

```diff
@@ -533,6 +533,12 @@ def mount_glusterfs(volume, target_path):
         return
 
     with mount_lock:
+        # Unmount to make sure no previous stale mounts
+        try:
+            execute(UNMOUNT_CMD, target_path)
```

Comment:
does it 'umount()' an active volume? i see mount_glusterfs() gets called from multiple places.

Reply:
coming here after `ismount` check. So should be fine I guess. But this I added before checking `ismount` not detecting stale mounts. We need to think about a different solution instead of using `ismount`